vsintegration/src/FSharp.Editor/Common/CancellableTasks.fs

```diff
@@ -0,0 +1,937 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
Probably need to add additional copyright for iced tasks (MIT), since a lot of code is pretty much reused from there.